mobius/graphs.py

```diff
@@ -40,6 +42,87 @@ def clear_buffer(num, path_buffer):
+        output_order_list = [1, 2, 3, 4, 5, 6]
+
+        if len(output) > 6:
+
+            for block_start_num in range(1, 7, 6):
+                """Adds first 6 graphs in order as is"""
+                keys = list(range(block_start_num, block_start_num + 6))
+                block = [output.get(key) for key in keys]
+                for num in range(len(block)):
+                    output_order.append(block[output_order_list[num]-1])
+
+            for block_start_num in range(7, len(output), 12):
+                if len(output) - block_start_num > 6:
+                    """For pages with 12 graphs on"""
+                    keys = list(range(block_start_num, block_start_num + 12))
+                else:
+                    """For last page if it only has 6 graphs on"""
+                    keys = list(range(block_start_num, block_start_num + 6))
+
+                block = [output.get(key) for key in keys]
+
+                graph_count = 0
+                order_values = []
+                original_order = list(range(0, 12))
+
+                for num in range(len(block)):
+                    """Rough way of getting block graphs in order. Scale y value larger as it is the most important"""
+                    graph_count += 1
+                    order_value = (round(float(str(block[num][2][0].start).split('+')[-1].split('j')[0]), -2) * 1000) + \
+                                  float((str(block[num][2][0].start).split('+')[0].split('(')[-1]))
+                    order_values.append(order_value)
+
+                new_order = [x for _, x in sorted(zip(order_values, original_order))]
+                output_order_list += [x + block_start_num for x in new_order]
+
+                for num in range(len(block)):
+                    """Reorder block based on new order"""
+                    output_order.append(block[new_order[num]])
+        else:
+            """Do nothing"""
+            output_order = output
+
+        if save:
+            """Renames SVG files accordingly if saved. Saves new plots to tmp folder, then deletes original svg
+            folder and renames tmp to svg.
+            """
+            os.mkdir(f"{output_folder}/tmp") if not os.path.exists(f"{output_folder}/tmp") else False
+            files = os.listdir(f"{output_folder}/svg")
+            files.sort(key=lambda f: int(re.sub('\D', '', f)))
+            num = 0
+            for file in files:
+                if file.endswith(".svg"):
+                    file.split('.')[0]
+
+                    new_file = f"{output_order_list.index(int(file.split('.')[0])) + 1}.svg"
+                    shutil.copyfile(f"{output_folder}/svg/{file}", f"{output_folder}/tmp/{new_file}")
+                    num += 1
+
+            shutil.rmtree(f"{output_folder}/svg")
+            os.rename(f"{output_folder}/tmp", f"{output_folder}/svg")
+
+        """Convert list to dictionary"""
+        output_order = convert_list_to_dict(output_order)
+
+        return output_order
```

Comment:
Think this needs to be:

```suggestion
        reordered_output = dict((output_order[key], value) for key, value in output.items())
        return reordered_output
```
to preserve the output type of the function dict[int, list[Path]]

Reply:
ignore this, think the problem is further up

Reply:
Shouldn't be needed with the latest commits